### PR TITLE
Feature/marquee component

### LINKED ISSUE #11: Marquee component
As a user,
I want makro-metrics,
so I can better assess the market.

- [x] Should show relevant makro-KPIs
- [x] Should animate from left to right
- [x] Should show total market-cap
- [x] Should show bitcoin-dominance
- [x] Should show ethereum-dominance
- [x] Should show 24h-change for each metric

### LINKED ISSUE #12: Dummy-data from assets-endpoint
As a developer,
I want mockup-data,
so I can build the marquee-component based on it.

- [x] Should contain Bitcoin- and Ethereum-Dominance in %
- [x] Should contain total market-cap
- [x] Should contain total trading-volume in 24h

### PR DESCRIPTION
**Pull request by konrascher3**
## Motivation
* Add marquee-component with macro-metrics
* Get macro-metrics from CoinGecko using a crawler
* Add Typography to marquee-content

## Issues closed
* closes #12 
* closes #11

### CODE REVIEW

**Comment by konrascher3** [comment]
Note: There are some bugs on mobile devices (Font Not rendering correctly; no auto-start)

- [ ] Fix font issue
- [ ] Fix auto-start

**Comment by konrascher3** [comment]
Font-bug may be caused be the Typography-Component from MUI and it's dynamic scaling

File: src/molecules/fastMarquee/fastMarquee.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const FastMarquee = () => {
+	return (
+		<Marquee
+			gradientWidth={10}
+			speed={25}
+		>
+			<div>
+				<MarqueeContent/>
+			</div>
```
Comment (pixelass) [inline src/molecules/fastMarquee/fastMarquee.jsx:14]: Is the additional `div` necessary here?

File: src/pages/index.jsx [hunk 2]
```diff
@@ -14,7 +16,7 @@ const columns = [
 		width: 0,
 		headerName: "#",
 		renderCell: (params) => (
-			/* TODO: Decide on  */
+			/* TODO: Decide on bookmark */
 			<div style={{ position: "absolute", inset: 0, display: "flex", alignItems: "center", justifyContent: "center" }}>
```
Comment (pixelass) [inline src/pages/index.jsx:20]: Too much inline-styles. You might want to move it to a styled-component.

**Review by pixelass — COMMENTED** [review]
Looks good, please look at my comments, they might help keeping the code maintainable and easier to read.

**Pull request merged**